css/properties/speak.json

```
@@ -9,7 +9,7 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80",
+              "version_added": "≤15",
```

Looks like it was added in Chrome 8: https://github.com/caugner/css-properties/blame/ddd65748134f1199c2f1e72a088098af072945d2/properties.css#L180

```suggestion
              "version_added": "8",
```